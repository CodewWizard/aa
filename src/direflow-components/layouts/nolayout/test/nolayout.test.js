import React from "react";
import renderer from "react-test-renderer";
import NoLayout from "../nolayout";

describe("NoLayout", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(
        <NoLayout>
          <div>Test Content</div>
        </NoLayout>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
