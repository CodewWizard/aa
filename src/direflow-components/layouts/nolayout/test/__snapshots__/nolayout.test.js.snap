// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoLayout renders correctly 1`] = `
<div>
  Test Content
</div>
`;
